pkg/operator/etc/config.go

```diff
@@ -27,6 +27,7 @@ type Config struct {
 	LeaderElectionEnabled                        bool          `env:"OPERATOR_LEADER_ELECTION_ENABLED" envDefault:"false"`
 	LeaderElectionID                             string        `env:"OPERATOR_LEADER_ELECTION_ID" envDefault:"starboard-lock"`
 	VulnerabilityScannerScanOnlyCurrentRevisions bool          `env:"OPERATOR_VULNERABILITY_SCANNER_SCAN_ONLY_CURRENT_REVISIONS" envDefault:"false"`
+	VulnerabilityScannerReportTTL                time.Duration `env:"OPERATOR_VULNERABILITY_SCANNER_REPORT_TTL" envDefault:"0s"`
```

Comment:
If we used pointer to `time.Duration` we'd skip default values and we can check if it's on or off with `Config.VulnerabilityScannerReportTTL != nil`

```suggestion
	VulnerabilityScannerReportTTL                *time.Duration `env:"OPERATOR_VULNERABILITY_SCANNER_REPORT_TTL"`
```